.circleci/config.base.yml

```diff
@@ -168,7 +176,7 @@ jobs:
       - run: chmod +x .circleci/lint_pr.sh && ./.circleci/lint_pr.sh
 
   verify-api-extract:
-    <<: *linux-e2e-executor-medium
+    <<: *linux-e2e-executor-xlarge
```

Comment:
this one is also CPU bound.

Reply:
this dropped from 2min 15 s to 54s.
however main driver of this particular change is that it started to be unstable on medium, i.e. hitting memory limit.